cpp/include/functions.h

```diff
@@ -15,6 +15,31 @@
  */
 #pragma once 
 
+/* ----------------------------------------------------------------------------*/
+
+/**
+ * @Synopsis Renumber source and destination indexes to be a dense numbering.
+ *
+ *    Assumptions:
+ *       * source and dest have same size and type
+ *       * source and dest are either GDF_INT32 or GDF_INT64
```

Comment:
Is it possible to provide GDF_INT64 as input and get GDF_INT32 on output if the dataset is small enough with this API? The cuda code seems to support it. If so, we should document it in the API.

Reply:
The current output will ALWAYS be GDF_INT32 in the python API, since only GDF_INT32 works in most of the algorithms.  Check the new text (including the new return value in case the renumbering doesn't fit in a 32-bit integer.